catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/feeds/FeedResource.java

```diff
@@ -142,9 +141,21 @@ public ThreadList list(
               description = "Filter threads by entity link",
               schema = @Schema(type = "string", example = "<E#/{entityType}/{entityFQN}/{fieldName}>"))
           @QueryParam("entityLink")
-          String entityLink)
+          String entityLink,
+      @Parameter(
+              description =
+                  "Filter threads by user id. This filter requires a 'filterType' query param. The default filter type is 'OWNER'",
+              schema = @Schema(type = "string"))
+          @QueryParam("userId")
+          String userId,
+      @Parameter(
+              description =
+                  "Filter type definition for the user filter. It can take one of 'OWNER', 'FOLLOWS', 'MENTIONS'. This must be used with the 'user' query param",
```

Comment:
Make this an enum and use the enum in the schema?

```
              schema = @Schema(implementation = FilterType.class))

```

Reply:
I have made the Schema change to the API as you suggested. It looks neat now. Thank you!